### PR TITLE
Add a guideline for contributors

### PR DESCRIPTION
**Pull request by sebastianpoeplau**
@aurelf these are the additions you suggested in #115.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Contributing to SymCC
+
+We encourage everyone to contribute improvements and bug fixes to SymCC. Our
+preferred way of accepting contributions is via GitHub pull requests. Please be
+sure to run clang-format on any C/C++ code you change, and ideally add a test to
```
Comment (aurelf) [inline CONTRIBUTING.md:5]: Good idea to suggest adding a test. Shall we also :
- (Obvious) tell people to check if the CI tests pass?  
- Add  a readme in the test folder to explain on how to add a test and run the test locally? Or an external documentation ?

**Review by aurelf — APPROVED** [review]
All looks good (+comment on test doc). Thanks !

File: CONTRIBUTING.md (hunk 1 above)
Comment (sebastianpoeplau) [reply CONTRIBUTING.md:5]: I've configured the repo to require passing tests, so the GitHub interface will tell people to fix them :blush: Regarding documentation, we have https://github.com/eurecom-s3/symcc/blob/master/docs/Testing.txt...

Comment (aurelf) [reply CONTRIBUTING.md:5]: Ah perfect I need to read that one :)

Comment (sebastianpoeplau) [reply CONTRIBUTING.md:5]: I've added a symlink to it from the `test` directory :wink:

**Pull request merged**